src/webhook_launcher/app/urls.py

```diff
@@ -25,7 +25,6 @@
 router = routers.DefaultRouter()
 router.register(r'webhookmappings', views.WebHookMappingViewSet)
 router.register(r'buildservices', views.BuildServiceViewSet)
-router.register(r'lastseenrevisions', views.LastSeenRevisionViewSet)
```

Comment:
It is not used by the webhook service and I figured it is better to remove it until it is needed and the LSR <-> WHM relation has been sorted out.